hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java

```diff
@@ -258,8 +258,13 @@ public void serverStarted(Id serverId, NodeRole serverRole) {
                  serverId, serverRole, this.name);
         this.serverInfoManager().initServerInfo(serverId, serverRole);
 
-        LOG.info("Search olap property key for graph '{}'", this.name);
-        this.schemaTransaction().initAndRegisterOlapTables();
+        // TODO: check necessary?
```

Comment:
we will rebuild missing olap table here (in another pr)

Reply:
Ok, when will the table be missing?

Reply:
at least 2 case:
1. create property key success, create table failed (with any reason)
2. the olap table store file delete/broken by some operations

and once it happens, we can't delete or search or recovery it, this problem will fixed in that pr